aexpect/remote.py

```diff
@@ -405,7 +404,7 @@ def remote_login(client, host, port, username, password, prompt, linesep="\n",
     elif client == "telnet":
         cmd = f"telnet -l {username} {host} {port}"
     elif client == "nc":
-        cmd = f"nc {verbose} {host} {port}"
+        cmd = f"nc {' '.join(extra_params)} {host} {port}"
```

Comment:
extra_params is used on 'ssh' and 'nc' now. However is not being used when using 'telnet'. If that is by design, then we should make it clear on the docstring.

Reply:
Sure

Reply:
@beraldoleal I updated the docstring, please have a look, thanks.